docs/src/pages/comparing-astro-vs-other-tools.md

```diff
@@ -0,0 +1,136 @@
+
+By contrast, Astro has built-in support for Partial Hydration and Island Architecture to only load the bare minimum amount of JavaScript in the browser. In production, Astro automatically optimizes your included component JavaSript bundles for you. This gives you built-in support to include individual JavaScript components on the page, without any effort on your part.
+
+#### Comparing Jekyll vs. Astro Performance
+
+Jekyll has few opinions about what you build, so it's difficult to compare generally. Because Jekyll includes zero JavaScript by default, it is possible for an Jekyll website to be as fast as an Astro website. However, because you are responsible for building your own build pipeline for any JavaScript that you include on the page, most Jekyll websites will not be as well-optimized as Astro websites.
+
+
+
+
+## Next.js vs. Astro
+
+[Next.js](https://nextjs.org/) is a popular website & application framework for React. Next.js uses React for HTML generation while Astro supports HTML, Astro components and most popular web frameworks (including React). 
+
+Next.js was designed to build any kind of website, but does best with highly dynamic websites (like Gmail or Facebook). Astro was also designed to build any kind of website, but does best with highly static websites (like content and eCommerce websites).
+
+Next.js supports both Static Site Generation (SSG) and Server-Side Rendering (SSR). Today, Astro only supports Static Site Generation (SSG).
+
+
+#### Comparing Next.js vs. Astro Performance
+
+In almost all cases, a website built with Astro will load significantly faster than a website built with Next.js. This is because Astro uses Partial Hydration and Island Architecture to only load the bare minimum amount of JavaScript in the browser. Next.js websites don't support this, and instead load your entire JavaScript application in the browser. The result is a slower page load and worse Lighthouse scores than a similar website built with Astro. 
+
+Next.js has [experimental support](https://piccalil.li/blog/new-year-new-website/#heading-no-client-side-react-code) for fully-static, zero-JavaScript pages. However, there is no planned support for hydrating individual components on the page. This leaves you with an all-or-nothing limitation.
+
+Next.js has great built-in image optimizations, which could make Next.js a better choice for some image-heavy websites.
+
+Case Study: [nextjs.org/docs](https://nextjs.org/docs/getting-started) (built with Next.js) loads **463kb** of JavaScript on first load, and scores a Lighthouse performance score of [59 out of 100](https://lighthouse-dot-webdotdevsite.appspot.com//lh/html?url=https%3A%2F%2Fnextjs.org%2Fdocs%2Fgetting-started). This website, (built with Astro) loads only **9.3kb** of JavaScript (98% less) and scores a Lighthouse performance score of [94 out of 100](https://lighthouse-dot-webdotdevsite.appspot.com//lh/html?url=https%3A%2F%2Fdocs.astro.build%2Fgetting-started). 
```

Comment:
This is a nice statistic, but I would qualify it with the fact that we don't have search or maybe some other nice features that nextjs docs might have setup since they've had more time to flesh out their docs 🤔

Reply:
good point, would comparing average speed load times, or using the google load metrics be a better way to convey the difference?

Reply:
Because lighthouse factors in a bunch of different points to give you your "overall performance" score, I can't think of a better single metric to summarize perf comparisons.

I agree that the "overall JS" number can come off as a bit cherry-picky. Maybe remove it and focus on the lighthouse score? It's pretty damning though... tbh I can't think of what would bump it up to almost half a MB just for a docs site. And that's gzipped! fwiw snowpack.dev added a pretty bloated search library, and that was just for ~40kb

Reply:
Yeah, I definitely think "look at the difference in JS shipped" is an effective argument, maybe it could be qualified with "most of nextjs's javascript is from shipping the entire site as JS, regardless of whether the content is interactive or not"?

Reply:
rewrote to be more clear! what do you think?

Reply:
I like it a lot! Thanks for being open to rewriting that bit!